### PR TITLE
Change to use params to store Vendor data in Apiary-Scenario.

### PR DESCRIPTION
**Pull request by martinconstante**
Creation of Asset Resource and update param asset request method.

### CODE REVIEW

File: examples/apiary-scenario/asset_accept_workflow.py [hunk 1]
```diff
@@ -44,24 +43,29 @@ class AssetAccept(AutomationEngine):
...
+                template_id = param.value
+        body = {"activation_tile": template_id}
+        response = self.approve(request.id, body)
         return response
 
+    def process_request(self, request):
+        if (request.type == 'purchase'):
+            for param in request.asset.params:
+                if (param.name == 'tenantId'):
+                    tenantIdParam = param.value
```
Comment (ffaraoneim) [inline examples/apiary-scenario/asset_accept_workflow.py:58]: Please change for snake case !

File: examples/apiary-scenario/asset_request_workflow.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        super().__init__(config=config)
+        self._fulfillment = Fulfillment(config)
+
+    def dispatch(self, request):
+        return self.process_request(request)
+
+    def process_request(self, request):
+        if (request.type == 'purchase'):
+            if (len(request.asset.items) == 1):
+                tenantIdParam = ''
```
Comment (ffaraoneim) [inline examples/apiary-scenario/asset_request_workflow.py:49]: snake case please!

File: examples/apiary-scenario/asset_accept_workflow.py (hunk 1 above)
Comment (martinconstante) [reply examples/apiary-scenario/asset_accept_workflow.py]: Fixed

File: examples/apiary-scenario/asset_request_workflow.py (hunk 2 above)
Comment (martinconstante) [reply examples/apiary-scenario/asset_request_workflow.py]: Fixed

**Pull request merged**